modules/git/git-v2.nu

```diff
@@ -130,7 +131,6 @@ export def gp [
         } else if $s.ahead > 0 {
             git push
         } else {
-            git fetch
         }
```

Comment:
i think this `else` is not needed anymore :relieved:

Reply:
ok, deleted

Reply:
thanks :blush: